src/geo/transform.js

```diff
@@ -203,12 +203,12 @@ class Transform {
     getVisibleUnwrappedCoordinates(tileID: CanonicalTileID) {
         const result = [new UnwrappedTileID(0, tileID)];
         if (this._renderWorldCopies) {
-            const utl = this.pointCoordinate(new Point(0, 0), 0);
-            const utr = this.pointCoordinate(new Point(this.width, 0), 0);
-            const ubl = this.pointCoordinate(new Point(this.width, this.height), 0);
-            const ubr = this.pointCoordinate(new Point(0, this.height), 0);
-            const w0 = Math.floor(Math.min(utl.column, utr.column, ubl.column, ubr.column));
-            const w1 = Math.floor(Math.max(utl.column, utr.column, ubl.column, ubr.column));
+            const utl = this.pointCoordinate(new Point(0, 0));
```

Comment:
Does `pointCoordinate` need to use `Point` for input? It could avoid the temporary object creation.